src/components/PeopplePage/index.ts

```diff
@@ -0,0 +1 @@
+export * from './PeoplePage';
```

Comment:
There is a typo in the folder name 'PeopplePage'. It should be 'PeoplePage' to maintain consistency with the file name 'PeoplePage'. 📝